configure.ac

```diff
@@ -365,6 +365,9 @@ AC_SUBST(PLATFORM_AOT_SUFFIX)
 
 ## PLATFORM_AOT_SUFFIX not so simple for windows :-)
 
+: ${CFLAGS="-g -O3"}
```

Comment:
I'm also not 100% sure whether the .deb that is used for benchmarking is built with CFLGS set so benchmarking results might not be meaningful.